tests/Builder/PipelineTestCase.php

```diff
@@ -22,11 +19,13 @@ final public static function assertSamePipeline(string|BackedEnum $expectedJson,
         }
 
         // BSON Documents doesn't support top-level arrays.
-        $expected = toPHP(fromJSON('{"root":' . $expectedJson . '}'))->root;
+        $expected = '{"pipeline":' . $expectedJson . '}';
```

Comment:
I realize this is outside of the PR diff, but when is a BackedEnum used to store the expected JSON string?

Reply:
Maybe @GromNaN has some new fancy ENUM ideas? 😆 🙈

Reply:
It was introduced in https://github.com/mongodb/mongo-php-builder/pull/6 to generate test pipelines from config Yaml.